extensions/interactions/customization-args-defs.ts

```diff
@@ -50,29 +58,49 @@ export interface ICodeReplCustomizationArgs {
     value: string;
   };
 }
+export interface ICodeReplCustomizationArgs extends
+    ICodeReplCustomizationArgsBackendDict { }
+
 
-export interface IContinueCustomizationArgs {
+export interface IContinueCustomizationArgsBackendDict {
   buttonText?: {
-    value: string;
+    value: ISubtitledUnicodeBackendDict;
+  };
+}
+export interface IContinueCustomizationArgs extends
+    Omit<IContinueCustomizationArgsBackendDict, 'buttonText'> {
```

Comment:
Instead of using Omit here, I suggest that you define this (and others) constructively (by specifying all fields), and then have a frontend test that iterates through all interaction types and validates that everything is consistent -- e.g. the backend dict interface and frontend object interface match (except for certain fields whose types need to be converted accordingly; this conversion should be expressible in a programmatic way based on the field type rather than on the field name).

Reply:
Would something like https://www.simonholywell.com/post/testing-typescript-types.html help? (Search for "dtslint" as well.)

That said, I think this can be deferred to a separate PR; if you elect to do this, please file an issue for that and clearly describe what needs to be done.

Reply:
Sure, I'll defer this for now